setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+npm install
+
+pushd .
+cd grassroots-frontend
+npm install
+popd
+
+pushd .
+cd grassroots-backend
+npm install
+popd
+
+# Instead of a traditional repo, we symlink shared code from the backend into the frontend.
+# Details on why are in the Readme.md.
```

Comment:
Those details aren't in the readme yet, but I'm not particularly worried about it.